Mathlib/Topology/Algebra/Module/StrongTopology.lean

```diff
@@ -102,6 +103,10 @@ instance instUniformSpace [UniformSpace F] [UniformAddGroup F]
     (by rw [UniformConvergenceCLM.instTopologicalSpace, UniformAddGroup.toUniformSpace_eq]; rfl)
 #align continuous_linear_map.strong_uniformity UniformConvergenceCLM.instUniformSpace
 
+theorem uniformSpace_eq [UniformSpace F] [UniformAddGroup F] (𝔖 : Set (Set E)) :
+    instUniformSpace σ F 𝔖 = UniformSpace.comap DFunLike.coe (UniformOnFun.uniformSpace E F 𝔖) := by
+  rw [instUniformSpace, UniformSpace.replaceTopology_eq]
```

Comment:
While you're at it, could you add the topology statement?

Reply:
done